app/screens/map/mapSelector.js

```diff
@@ -29,11 +30,33 @@ function postalCodeFilterSelector(state) {
   return state.filters.postalCodes;
 }
 
+const cityUnitsSelector = createSelector(
```

Comment:
- [ ] There starts to be some repeating selectors, for example the `cityUnitsSelector`. These could be moved to for example `state/selectors/` folder. 

Benefits:
- No repeated code.
- Easier to test when the selector is in a separate file.
- if state at some point changes it's easier to change the selector as well.